Tool/src/packaging/windows/build_zip_win.sh

```diff
@@ -3,20 +3,8 @@ echo "****************************************"
 echo "Creating zip file for Windows amd64"
 echo "****************************************"
 
-BUILD_FOLDER=${BGO_SPACE}/build/xray
+DIST_FOLDER=${BGO_SPACE}/build/dist/
+cd $DIST_FOLDER
 
-echo "Constructing the zip package"
-
-if [ -f ${BUILD_FOLDER}/aws-xray-daemon-windows-service-`cat ${BGO_SPACE}/VERSION`.zip ]
-then
-    rm ${BUILD_FOLDER}/aws-xray-daemon-windows-service-`cat ${BGO_SPACE}/VERSION`.zip
-fi
-
-if [ -f ${BUILD_FOLDER}/aws-xray-daemon-windows-process-`cat ${BGO_SPACE}/VERSION`.zip ]
-then
-    rm ${BUILD_FOLDER}/aws-xray-daemon-windows-process-`cat ${BGO_SPACE}/VERSION`.zip
-fi
-
-cd ${BUILD_FOLDER}
-zip aws-xray-daemon-windows-service-`cat ${BGO_SPACE}/VERSION`.zip xray.exe cfg.yaml LICENSE THIRD-PARTY-LICENSES.txt
-zip aws-xray-daemon-windows-process-`cat ${BGO_SPACE}/VERSION`.zip xray_windows.exe cfg.yaml LICENSE THIRD-PARTY-LICENSES.txt
+zip aws-xray-daemon-windows-amd64-service-${VERSION}.zip ../xray-windows-amd64/xray_service.exe ../xray/cfg.yaml ../xray/LICENSE ../xray/THIRD-PARTY-LICENSES.txt
+zip aws-xray-daemon-windows-amd64-${VERSION}.zip ../xray-windows-amd64/xray.exe ../xray/cfg.yaml ../xray/LICENSE ../xray/THIRD-PARTY-LICENSES.txt
```

Comment:
Changing the zip names to include the arch is potentially kinda-sorta breaking because we have example scripts that reference the `3.x` ZIP file in S3 with the current naming format, e.g. https://docs.aws.amazon.com/xray/latest/devguide/xray-daemon-local.html#xray-daemon-local-windows. 

We could:
1. Keep naming as is here and in S3
2. Update the naming here, but keep the name the same for the 3.x ZIP in S3
3. Change name of zip files here and in S3 (possibly breaking)

If we're blowing up the binaries and such anyway on this release, then we might as well go with option 3. It's just going to be a big documentation update to the effect of "for version 3.2.0 and earlier do things this way and for newer versions do things the other way"

Reply:
I would suggest making these breaking changes with a 4.x version of the daemon with detailed instructions in our doc as Will said, so that the customers using the current links wouldn't expect to get 3.3.0 as the links would only work till 3.2.0.

Reply:
If we want to provide some transition time, we can upload duplicate zip files with the old and new names right?

Reply:
Discussed offline, we will move forward with duplicate ZIPs, update our docs to reference the new ZIP names, but still keep the old names up-to-date until the 4.x major version bump to keep backwards-compatibility.